compiam/melody/pattern/sancara_search/extraction/pitch.py

```diff
@@ -0,0 +1,108 @@
+import math
+import numpy as np
+import pandas as pd 
+
+import librosa
+
+from spleeter.separator import Separator
+from spleeter.audio.adapter import AudioAdapter
+
+import essentia.standard as estd
+
+from scipy.ndimage import gaussian_filter1d
+
+from compiam.melody.pattern.sancara_search.extraction.sequence import get_stability_mask, add_center_to_mask
+from compiam.melody.pattern.sancara_search.extraction.io import get_timeseries, write_timeseries
+
+def pitch_to_cents(p, tonic):
+    """
+    Convert pitch value, <p> to cents above <tonic>.
+
+    :param p: Pitch value in Hz
+    :type p: float
+    :param tonic: Tonic value in Hz
+    :type tonic: float
+
+    :return: Pitch value, <p> in cents above <tonic>
+    :rtype: float
+    """
+    return 1200*math.log(p/tonic, 2) if p else None
```

Comment:
same here: todo esto podría estar en `compiam/utils/pitch.py`. We can leave that for this other PR though.